src/interaction/TrackManager.ts

```diff
@@ -614,4 +636,19 @@ export default class TrackManager<POIMeta> {
     this.source_.changed();
     this.shadowTrackLayer_.getSource().changed();
   }
+
+  private calculateMaskFeature(extent: Extent) {
+    const projection = this.map_.getView().getProjection();
+    // for some projections (like EPSG:2056) extent smaller than visible map
+    let wExtent = projection.getExtent();
+    if (projection.getUnits() === 'm') {
+      wExtent = epsg3857Extent;
+    } else if (projection.getUnits() === 'degrees') {
+      wExtent = epsg4326Extent;
+    }
+    const mask = fromExtent(wExtent);
+    const drawingArea = fromExtent(extent);
+    mask.appendLinearRing(drawingArea.getLinearRing(0));
+    return new Feature(mask);
+  }
```

Comment:
I don't understand why you hardcode these 2 projections.

Reply:
because otherwise, the mask can be smaller than the canvas size, for example for EPSG:2056 projection:
![image](https://github.com/user-attachments/assets/c0bf9f19-d13b-4e39-9168-0d71871f8b92)


Probably it should be done for all possible units, but I take two most common

Reply:
Also, it is possible to use the size from the current view but then we will need to update polygon geometry on each map move, and it will be probably worse in terms of performance

Reply:
Look how it is done for print mask in newui:
-  this.map.on('postcompose', postcompose)
- and the function in src/helpers/print.ts

Reply:
updated, thanks